docs/components/guidesXml.rst

```diff
@@ -0,0 +1,71 @@
+
+..  _guides-xml:
+
+==========
+guides.xml
+==========
+
+..  todo: Add general docs about guides.xml
+
+..  _guides-xml-add-config-extension:
+
+Add a configuration option (Extension)
+======================================
+
+Extension authors can add additional configuration options in the
+extension's tag:
```

Comment:
```suggestion
:xml:`extension` tag (allowed to occur multiple times):
```

Something like that to indicate it's not a single tag only, and to attribute it to "xml" and not "your extension".